src/test/java/org/springframework/integration/aws/outbound/SnsMessageHandlerTests.java

```diff
@@ -99,9 +99,10 @@ void testSnsMessageHandler() {
 
 		Map<String, MessageAttributeValue> messageAttributes = publishRequest.messageAttributes();
 
-		assertThat(messageAttributes).doesNotContainKey(MessageHeaders.ID);
-		assertThat(messageAttributes).doesNotContainKey(MessageHeaders.TIMESTAMP);
-		assertThat(messageAttributes).containsKey("foo");
+		assertThat(messageAttributes)
+				.doesNotContainKey(MessageHeaders.ID)
+				.doesNotContainKey(MessageHeaders.TIMESTAMP)
```

Comment:
Looks like `doesNotContainKeys()` can be used instead.

Reply:
Will apply!